docs/USAGE-GUIDE.md

```diff
@@ -669,6 +669,18 @@ callback function takes three arguments: a pointer to abitrary data for use
 within the callback, a pointer to a key which can be used to delete the
 cached entry, and a 64 bit unsigned integer specifying the size of this key.
 
+### s2n\_config\_set\_max\_fragment\_length
```

Comment:
For our two API functions what about:

- `s2n_config_send_max_fragment_length(config, max_frag_len)` for the client
- `s2n_config_accept_max_fragment_length(config)` for the server.

I think a user who is savvy enough to want to use MFL can infer that the "send" function only applies to client mode and the "accept" function only applies to server mode. I don't think it's worth taxing our minds to produce a unified client/server API here. The extension assumes asymmetric capability between client/server.